plugins/com.google.cloud.tools.eclipse.appengine.libraries.test/src/com/google/cloud/tools/eclipse/appengine/libraries/model/CloudLibrariesInPluginXmlTest.java

```diff
@@ -114,7 +114,7 @@ public void testEndpointsLibraryConfig() throws URISyntaxException {
     assertThat(mavenCoordinates.getRepository(), is("central"));
     assertThat(mavenCoordinates.getGroupId(), is("com.google.endpoints"));
     assertThat(mavenCoordinates.getArtifactId(), is("endpoints-framework"));
-    assertThat(mavenCoordinates.getVersion(), is("2.0.10"));
+    assertThat(mavenCoordinates.getVersion(), is("2.0.11"));
```

Comment:
we probably should also change this so that it's 2.0.11 or later; but this is a good quick fix